### PR TITLE
DATAUP-591: normalise casing in display.yaml files

### PR DESCRIPTION
**Pull request by ialarmedalien**
Standardised casing to sentence case in all the ui-name fields.
Standardised casing on `GenBank`.
Upped version number.

# Jira Ticket / Issue

e.g. https://kbase-jira.atlassian.net/browse/DATAUP-591

-   [x] Added the Jira Ticket to the title of the PR e.g. (DATAUP-69 Adds a PR template)

# Testing Instructions

-   Details for how to test the PR: 
-   [ ] Tests pass in Travis-CI and locally 

# Dev Checklist:

-   [x] My code follows the guidelines at <https://sites.google.com/truss.works/kbasetruss/development>
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [N/A] I have made corresponding changes to the documentation, including updating the README with app information changes
-   [x] My changes generate no new warnings
-   [N/A] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing tests pass locally with my changes

# Updating Version and Release Notes (if applicable)

-   [x] [Version has been bumped](https://semver.org/) in `kbase.yml`
-   [x] [Release notes](/RELEASE_NOTES.md) have been updated for each release (and during the merge of feature branches)

### CODE REVIEW

File: ui/narrative/methods/batch_import_assembly_from_staging/display.yaml [hunk 1]
```diff
@@ -21,30 +21,28 @@ suggestions :
...
+        ui-name : Assembly type
         short-hint : |
             The type or source of this assembly
         long-hint  : |
             The type or source of this assembly
 
     min_contig_length :
-        ui-name : |
-            Minimum Contig Length
+        ui-name : Minimum Contig Length
```
Comment (MrCreosote) [inline ui/narrative/methods/batch_import_assembly_from_staging/display.yaml:41]: not sentence case

File: ui/narrative/methods/import_attribute_mapping_from_staging/display.yaml [hunk 2]
```diff
@@ -21,16 +21,14 @@ suggestions :
...
+        ui-name : File path
         short-hint : |
             File with tabular Attribute Mapping data.
         long-hint  : |
             Recognized extensions for the input file are .tsv or .xlsx
 
     attribute_mapping_name :
-        ui-name : |
-            Attribute Mapping object name
+        ui-name : Attribute Mapping object name
```
Comment (MrCreosote) [inline ui/narrative/methods/import_attribute_mapping_from_staging/display.yaml:31]: Not sure if mapping should be capitalized or not

File: ui/narrative/methods/import_tsv_as_expression_matrix_from_staging/display.yaml [hunk 3]
```diff
@@ -21,16 +21,14 @@ suggestions :
...
+        ui-name : TSV file path
         short-hint : |
             File with expression data in TSV format with Features in rows and Conditions in columns
         long-hint  : |
             The filename should end with ".tsv"
 
     matrix_name :
-        ui-name : |
-            Expression Matrix Object Name
+        ui-name : Expression Matrix object name
```
Comment (MrCreosote) [inline ui/narrative/methods/import_tsv_as_expression_matrix_from_staging/display.yaml:31]: Does matrix need to be capitalized?

File: ui/narrative/methods/import_tsv_as_phenotype_set_from_staging/display.yaml [hunk 4]
```diff
@@ -21,24 +21,21 @@ suggestions :
...
+        ui-name : Phenotype TSV file path
         short-hint : |
             Select a phenotype file in TSV format
         long-hint  : |
             Filename should end in .tsv
 
     phenotype_set_name :
-        ui-name : |
-            Phenotype Set Name
+        ui-name : Phenotype Set name
```
Comment (MrCreosote) [inline ui/narrative/methods/import_tsv_as_phenotype_set_from_staging/display.yaml:31]: Set?

**Review by MrCreosote — CHANGES_REQUESTED** [review]
Tests aren't passing - they'll need to be fixed

Otherwise just some minor stuff

File: ui/narrative/methods/batch_import_assembly_from_staging/display.yaml (hunk 1 above)
Comment (ialarmedalien) [reply ui/narrative/methods/batch_import_assembly_from_staging/display.yaml:41]: Good catch, thank you!

File: ui/narrative/methods/import_attribute_mapping_from_staging/display.yaml (hunk 2 above)
Comment (ialarmedalien) [reply ui/narrative/methods/import_attribute_mapping_from_staging/display.yaml:31]: There seemed to be capitalisation of KBase types, so I left that as-is; similarly 'Narrative' is capitalised all over the place, which is obviously _deeply_ wrong, but I let that go.

File: ui/narrative/methods/import_tsv_as_expression_matrix_from_staging/display.yaml (hunk 3 above)
Comment (ialarmedalien) [reply ui/narrative/methods/import_tsv_as_expression_matrix_from_staging/display.yaml:31]: apparently so if it is an Expression Matrix

File: ui/narrative/methods/import_tsv_as_phenotype_set_from_staging/display.yaml (hunk 4 above)
Comment (ialarmedalien) [reply ui/narrative/methods/import_tsv_as_phenotype_set_from_staging/display.yaml:31]: it isn't just a set, it's a Phenotype Set!

**Review by MrCreosote — APPROVED** [review]
Approved contingent on passing tests

**Review by Tianhao-Gu — APPROVED** [review]
No implementation code changes. I am 👍  to merge and ignore the failing tests.

**Pull request merged**